packages/graph-engine/src/utils/node.ts

```diff
@@ -12,3 +12,22 @@ export const getAllOutputs = <T extends Node>(node: T) => {
 		Object.entries(node.outputs).map(([key, value]) => [key, value.value])
 	) as UnwrapOutput<T['outputs']>;
 };
+
+export function cloneInnerGraph(originalNode: Node, clonedNode: Node) {
```

Comment:
I'm not sure about this, this is again assuming that there will always be a single graph on a node with that exact name, why not amend the interface to return the names of the properties and then copy them across if you're going to do this ? Also don't use any typing here , you shouldn't need to , it should be a node that extends the interface